core/domain/feedback_services.py

```diff
@@ -285,10 +281,9 @@ def get_messages(thread_id):
     Returns:
         list(FeedbackMessage). Contains all the messages in the thread.
     """
-    return [
-        _get_message_from_model(m)
-        for m in feedback_models.GeneralFeedbackMessageModel.get_messages(
-            thread_id)]
+    return [_get_message_from_model(m)
```

Comment:
As an FYI, we generally prefer the previous indentation (break after "[" rather than hanging alignment since the latter tends to squeeze things to the right). It's not a strong rule though, so your call.

Reply:
Acknowledged.